src/worldmap.ts

```diff
@@ -181,14 +201,14 @@ export default class WorldMap {
   }
 
   calcCircleSize(dataPointValue) {
-    const circleMinSize = parseInt(this.ctrl.panel.circleMinSize, 10) || 2;
+    let circleMinSize = parseInt(this.ctrl.panel.circleMinSize, 10) || 2;
     const circleMaxSize = parseInt(this.ctrl.panel.circleMaxSize, 10) || 30;
 
-    if (this.ctrl.data.valueRange === 0) {
+    if (this.valueRange === 0) {
       return circleMaxSize;
     }
 
-    const dataFactor = (dataPointValue - this.ctrl.data.lowestValue) / this.ctrl.data.valueRange;
+    let dataFactor = (dataPointValue - this.lowestValue) / this.valueRange;
```

Comment:
Why is it `let`?

Reply:
Same as the let on circleMinSize... obsolete left behind.